src/core/curves.js

```diff
@@ -340,15 +340,11 @@ p5.prototype.curve = function() {
   if (!this._renderer._doStroke) {
     return this;
   }
-  if (this._renderer.isP3D){
+  if (this._renderer.isP3D) {
     args.push(curveDetail);
     this._renderer.curve(args);
-  } else{
-    this._renderer.curve(
-      args[0], args[1],
-      args[2], args[3],
-      args[4], args[5],
-      args[6], args[7]);
+  } else {
+    this._renderer.curve(args[0], args[1], args[2], args[3], args[4], args[5], args[6], args[7]);
```

Comment:
There are many lines like this that are rather long. We used to enforce 80 character limit per line with jscs, I'm not sure if eslint does that right now but I think it is good to keep otherwise you may get some lines that are hard to read on a text editor because you need to scroll sideways and also hard to view on git commits.

Reply:
Alright, I'll go back to 80 characters. To me 100 characters looked better in my editor, but I get your points.